src/ape/managers/chain.py

```diff
@@ -482,7 +482,14 @@ def restore(self, snapshot_id: Optional[SnapshotID] = None):
         self.provider.revert(snapshot_id)
         self.account_history.revert_to_block(self.blocks.height)
 
-    def mine(self, num_blocks: int = 1, timestamp: Optional[int] = None) -> None:
+    def mine(
+        self,
+        num_blocks: int = 1,
+        timestamp: Optional[int] = None,
+        deltatime: Optional[int] = None,
+    ) -> None:
         if timestamp:
             self.pending_timestamp = timestamp
+        if deltatime:
```

Comment:
are both `timstamp` and `deltatime` arguments allowed together?

Reply:
needs to be elif, and we need to choose which overrides the other